e2e/cmd/replay/command.go

```diff
@@ -4,25 +4,35 @@ import (
 	"errors"
 	"flag"
 	"fmt"
-	"log"
 	"strings"
 	"sync"
 	"time"
 
+	"github.com/0xPolygon/pbft-consensus/e2e/helper"
+
+	"github.com/mitchellh/cli"
+
 	"github.com/0xPolygon/pbft-consensus"
 	"github.com/0xPolygon/pbft-consensus/e2e"
-	"github.com/mitchellh/cli"
+	"github.com/0xPolygon/pbft-consensus/e2e/replay"
 )
 
-// ReplayMessageCommand is a struct containing data for running replay-message command
-type ReplayMessageCommand struct {
+// MessageCommand is a struct containing data for running replay-message command
+type MessageCommand struct {
```

Comment:
`replay` is the package name. Based on the most common go codestyle, you should avoid using the name of a pacakge in the types name. Like it is better to use `replay.Command` instead of `replay.ReplayCommand`. No needed to duplicate names. 

Updated to just `Command`.

Reply:
Agreed.